src/api/app/views/webui2/webui/repositories/_breadcrumb_items.html.haml

```diff
@@ -5,7 +5,7 @@
 - if current_page?(repositories_path)
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
     Repositories
-- elsif current_page?(project_repository_state_path)
+- elsif defined?(@repository) && current_page?(project_repository_state_path)
```

Comment:
I count that as vote for #6472 :)

Reply:
we use instance variables in the breadcrumb. I think that is ok, but I would say this is an special case.

Reply:
You take the one exception from the rule (that we consciously took) and use it as a counter-argument? Nice try:smiley: